lib/travis/api/v3/renderer/owner.rb

```diff
@@ -19,5 +19,10 @@ def repositories
       repositories = query(:repositories).for_owner(@model)
       access_control.visible_repositories(repositories)
     end
+
+    def github_installation_id
+      installation = query(:installation).for_owner(@model).first
+      installation.github_id if installation
+    end
```

Comment:
I'm not sure why this has to be a field on owner. I think it should be a field on installation, and then web could include it via `include=owner.installation`.